osbs/conf.py

```diff
@@ -286,6 +286,10 @@ def get_source_registry_uri(self):
     def get_pulp_registry(self):
         return self._get_value("pulp_registry_name", self.conf_section, "pulp_registry_name")
 
+    def get_pulp_manifest_lists(self):
+        return self._get_value("pulp_manifest_lists", self.conf_section,
```

Comment:
as I understand it, we only use manifest lists in atomic reactor if the pulp registry supports them. I'd like to keep pulp in the name to remind people that this is a pulp configuration issue.